From 3593df4c96955b69485acbc0b7fa08569fe6b8d8 Mon Sep 17 00:00:00 2001
From: Jani Hautakangas <jani@kodegood.com>
Date: Mon, 1 Sep 2025 22:12:44 +0300
Subject: [PATCH 2/2] Disable SRT_ENABLE_BINDTODEVICE for OHOS

SRT_ENABLE_BINDTODEVICE causes linux/if.h to
include linux/socket.h and after srt includes
sys/socket.h which causes redefinition of
sockaddr_storage and thus an error
---
 CMakeLists.txt | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/CMakeLists.txt b/CMakeLists.txt
index 1492306a..acaef5c6 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -737,10 +737,10 @@ add_definitions(
 	-DSRT_VERSION="${SRT_VERSION}"
 )
 
-if (LINUX)
+# if (LINUX)
 # This is an option supported only on Linux
-	add_definitions(-DSRT_ENABLE_BINDTODEVICE)
-endif()
+	# add_definitions(-DSRT_ENABLE_BINDTODEVICE)
+# endif()
 
 # This is obligatory include directory for all targets. This is only
 # for private headers. Installable headers should be exclusively used DIRECTLY.
-- 
2.48.1

